shell/platform/windows/flutter_windows_winuwp.cc

```diff
@@ -25,9 +29,22 @@ static flutter::FlutterWindowsEngine* EngineFromHandle(
   return reinterpret_cast<flutter::FlutterWindowsEngine*>(ref);
 }
 
+// Resturns a list of discrete arguments splitting the input using a ",".
+std::vector<std::string> split(std::string const s) {
```

Comment:
Nit: `Split`, per Google style. (Or preferably, `SplitCommaSeparatedString` for clarity about what exactly it's doing).